### PR TITLE
docs: add policy information in reference docs

### PR DESCRIPTION
**Pull request by migmartri**
- updates contract reference with the rest of materials, plus policies
- Expands the reference guide section
- Updates project name
- Fixes broken link in readme 
- Updates dagger info

### CODE REVIEW

File: docs/docs/reference/operator/contract.mdx [hunk 1]
```diff
@@ -43,6 +43,7 @@ A full example of a Workflow Contract looks like
 | `envAllowList`                  | no       | List of environment variables that will be resolved and injected in the attestation                                                                                          |
 | [`runner`](#runner-context)     | no       | Specific runner type associated with this contract. If not set, this contract will be valid to be run `anywhere` but you'll miss out some of [its benefits](#runner-context) |
 | `annotations`                   | no       | Name/Value pairs of arbitrary annotations that will be added to the attestation. **If the value is not provided, it will be required during the attestation process.**       |
+| [`policies`](#policy-attachments)                   | no       | Attachments to existing Chainloop policies. See [policies reference guide](http://localhost:3000/reference/policies) for more information       |
```
Comment (javirln) [inline docs/docs/reference/operator/contract.mdx:46]: This link might be wrong

Comment (migmartri) [reply docs/docs/reference/operator/contract.mdx:46]: It's a section at the bottom of the page.

Comment (javirln) [reply docs/docs/reference/operator/contract.mdx:46]: What I mean is that it's pointing to `localhost:3000` is that intended?

Comment (migmartri) [reply docs/docs/reference/operator/contract.mdx:46]: ahhhh, sorry, you are right, I was looking at [`policies`](#policy-attachments)

Comment (migmartri) [reply docs/docs/reference/operator/contract.mdx]: Fixed, thanks for catching it

**Review by javirln — APPROVED** [review]

**Pull request merged**